src/features/home/ui/TopFiveStacks.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Meta, StoryObj } from '@storybook/react';
+import TopFiveStacks from './TopFiveStacks';
+
+const meta: Meta<typeof TopFiveStacks> = {
+  title: 'component/home/TopFiveStacks',
```

Comment:
스토리의 title 컨벤션이 다 다른 것 같은데 확인 한번 부탁드립니다~

원준님은 이런 느낌으로 쓰시고 계신 것 같네요!
```suggestion
  title: 'components/features/TopFiveStacks',
```

Reply:
#89 에도 코멘트 드렸듯이, components/해당하는 레이어명/컴포넌트 명 으로 사용중입니다!

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/87/commits/6891fdb4d37555b00cba96ea9de121d15c13f721 에서 반영했습니다!!